api_server/src/main/java/io/zetch/app/domain/Restaurant.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Restaurant entity definition
+ */
+package io.zetch.app.domain;
+
+import javax.persistence.*;
+
+@Entity
+public class Restaurant {
+
+    @Id
+    @GeneratedValue
+    private Integer id;
```

Comment:
Generally speaking, it's better to make it `Long` as our users can be more than `2,147,483,647`. 🤣